QUARKUS-6244.MD

```diff
@@ -0,0 +1,83 @@
+# QUARKUS-6244 - Add Jakarta Data documentation, dependency management and tests
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6244
+
+Upstream issues: https://github.com/quarkusio/quarkus/issues/44278, https://github.com/quarkusio/quarkus/issues/35603
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/47012, https://github.com/quarkusio/quarkus-quickstarts/pull/1516
+
+Starting with Quarkus 3.22, Quarkus documents support for Jakarta Data 1.0.1 when used with Hibernate ORM extension.
+This feature relies heavily on the `hibernate-jpamodelgen` annotation processor and is implemented almost entirely outside the Quarkus.
+Quarkus itself plans to integrate Jakarta Data support through Panache extension and complement the active record pattern.
+The QUARKUS-6244 itself is about vanilla Hibernate ORM support for Jakarta Data 1.0.1.
+Jakarta Data goals are simplifying data access and their manipulation (e.g. by making sessions stateless), unify API (Jakarta NoSQL and Persistence specs), abstracting layer common for different databases and storage sources and so on.
+That means that we need to test all supported databases.
+We need to verify integration points between Quarkus and Hibernate ORM as the vast majority of upstream test coverage covers vanilla Hibernate ORM.
+There is no clear connection of this feature to OpenShift or any other environment, therefore we just run a smoke test.
+
+## Scope of the testing
+
+Testing must focus on Jakarta Data integration with Quarkus for supported configurations and environments:
+
+- all functionalities shall be tested for PostgreSQL, MariaDB, Oracle, SQL Server and DB2
```

Comment:
What about MySQL? It's also [supported]((https://access.redhat.com/articles/4966181) ) + it's kinda popular.